process/logsevents/delegatorsProcessor.go

```diff
@@ -68,12 +68,17 @@ func (dp *delegatorsProc) processEvent(args *argsProcessEvent) argOutputProcessE
 	// topics[1] = active stake
 	// topics[2] = num contract users
 	// topics[3] = total contract active stake
-	// topics[4] = true - if delegator was deleted in case of withdrawal
+	// topics[4] = true - if delegator was deleted in case of withdrawal OR contract address in case of delegate from async
```

Comment:
what is delegate from async?

Reply:
When we do `delegate` though this operations 
`makeNewContractFromValidatorData` or
`mergeValidatorToDelegationSameOwner` or
`mergeValidatorToDelegationWithWhitelist`

Reply:
still, that is not "async"
better:
```GO
// topics[4] = true - if the delegator was deleted in case of withdrawal OR the contract address in case of delegate operations from staking v3.5 (makeNewContractFromValidatorData, mergeValidatorToDelegationSameOwner or mergeValidatorToDelegationWithWhitelist)
```